src/Test/View.elm

```diff
@@ -0,0 +1,64 @@
+module Test.View
+    exposing
+        ( find
+        , findAll
+        , trigger
+        , has
+        , count
```

Comment:
What's the benefit of duplicating the `Html.Query` API vs just having a `TestContext -> Query.Single` function?